### PR TITLE
fix(#374) - Add base_url for notification SPA

### LINKED ISSUE #374: Add base_url for notification SPA
## Describe the bug

- After deploying on SPAShip the assets failed to load because of the route.

## Expected behavior

Should have the correct base_url after it has been deployed

## Screenshots

Screenshot 2020-06-04 at 4 40 22 PM

## Environment

- Operating System(s): macOS
- Browser: Firefox/Chrome

### PR DESCRIPTION
**Pull request by hybridx**
fix(#374) - Added notifications to notification-spa base_url

### Resolves #374 

# Explain the feature/fix

Added notification to the base_url

## Does this PR introduce a breaking change

No

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?

### CODE REVIEW

File: packages/notification-spa/package.json [hunk 1]
```diff
@@ -12,45 +12,45 @@
...
-    "ts-jest": "26.1.0",
-    "tslib": "2.0.0",
-    "zone.js": "0.10.3"
+    "graphql-tag": "2.10.0",
+    "jest": "25.4.0",
+    "jest-preset-angular": "8.1.3",
+    "rxjs": "6.5.4",
+    "ts-jest": "25.4.0",
+    "tslib": "1.10.0",
+    "zone.js": "0.10.2"
```
Comment (riginoommen) [inline packages/notification-spa/package.json:39]: Why most of the packages are downgraded?

File: packages/notification-spa/package.json [hunk 2]
```diff
@@ -12,45 +12,45 @@
...
-    "jasmine-spec-reporter": "5.0.2",
-    "protractor": "7.0.0",
-    "ts-node": "8.10.2",
-    "tslint": "5.20.1",
-    "typescript": "3.9.3"
+    "jasmine-spec-reporter": "4.2.1",
+    "protractor": "5.4.3",
+    "ts-node": "8.3.0",
+    "tslint": "5.18.0",
+    "typescript": "3.7.5"
```
Comment (riginoommen) [inline packages/notification-spa/package.json:54]: Why most of the packages are downgraded?

File: packages/notification-spa/package.json (hunk 1 above)
Comment (hybridx) [reply packages/notification-spa/package.json:39]: Removed the unwanted package downgrades. And changed the verison for typescript to 8.3.8 as angular only support ts < 3.9.0

Screenshot 2020-06-04 at 7 42 18 PM

File: packages/notification-spa/package.json (hunk 2 above)
Comment (hybridx) [reply packages/notification-spa/package.json:54]: Removed the unwanted package downgrades. And changed the verison for typescript to 8.3.8 as angular only support ts < 3.9.0

Screenshot 2020-06-04 at 7 42 18 PM

File: packages/notification-spa/package.json [hunk 3]
```diff
@@ -51,6 +51,6 @@
     "protractor": "7.0.0",
     "ts-node": "8.10.2",
     "tslint": "5.20.1",
-    "typescript": "3.9.3"
+    "typescript": "3.8.3"
```
Comment (riginoommen) [inline packages/notification-spa/package.json:54]: Can you downgrade to 3.9.0 as shown in the console

File: packages/notification-spa/package.json (hunk 2 above)
Comment (riginoommen) [reply packages/notification-spa/package.json]: Okay i understand the dependency of the typescript. But make sure that typescript should be at most updated version like as i saw in message 3.9.0

Comment (hybridx) [reply packages/notification-spa/package.json]: It shows less than 3.9.0. I did try installing 3.9.0 it still throws an error

**Pull request merged**